custom_components/google_fit/const.py

```diff
@@ -77,6 +80,16 @@
         source="derived:com.google.calories.expended:com.google.android.gms:merge_calories_expended",  # pylint: disable=line-too-long
         data_key="calories",
     ),
+    GoogleFitSensorDescription(
+        key="google_fit",
+        name="Basal Metabolic Rate",
+        icon="mdi:target",
+        native_unit_of_measurement="kcal",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=None,
+        source="derived:com.google.calories.bmr:com.google.android.gms:merged",
+        data_key="basalMetabolicRate",
```

Comment:
You should also add this and the other new sensors to the sensor table in the README at the root of the repo

Reply:
Done : https://github.com/YorkshireIoT/ha-google-fit/pull/74/commits/6ca1a47f4b66710f4386dc71d0e5ec83109bbd06

Reply:
You still haven't added basal metabolic rate to the README

Reply:
Done https://github.com/YorkshireIoT/ha-google-fit/pull/74/commits/1318b9b7f971cf9b02c1b730210bd9c61309bff8